src/keyszer/output.py

```diff
@@ -36,6 +36,11 @@
 _uinput = None
 
 
+# for use with throttle delays
+def sleep_ms(msec):
+    return time.sleep(msec / 1000)
```

Comment:
```suggestion
def sleep_ms(msec):
    if msec == 0:
        return
    return time.sleep(msec / 1000)
```

Reply:
I guess I'm not the only one prone to fits of "Oh, no, is this a divide-by-zero that I need to protect against, no wait, it's a harmless zero-divided-by." Heh.

Reply:
`sleep(0)` is a short sleep because it hands control back to the OS - unless the function itself protects against this - and often they don't on purpose to allow for short context switches.